build-tools/create-pkg/distribution.xml.in

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<installer-gui-script minSpecVersion="1">
+    <license file="License" mime-type="application/rtf" />
```

Comment:
We should reformat this file to follow our XML indentation convention used elsewhere, e.g. 2 space indents.